.github/dependabot.yml

```diff
@@ -0,0 +1,32 @@
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
+      commit-message:
+      prefix: ":seedling:"
```

Comment:
I think prefix should be indented one more level (same as line 28).

Reply:
thanks, actually the prvious line was wrongly indented